src/aem/azext_aem/custom.py

```diff
@@ -265,11 +265,9 @@ def _create_role_assignments_for_scopes(self, scopes):
 
                     self._roles_client.role_assignments.create(scope, assignment_name, params_role_assignment)
                     created = True
-                except CloudError as cex:
+                except HttpResponseError as cex:
                     logger.info("Error during role assignment %s", cex)
-                    if ((not cex.error) or (not cex.error.error) or
-                            (PRINCIPAL_NOT_FOUND_ERROR != cex.error.error.lower())):
-                        raise
+                    raise
```

Comment:
The original logic is to silence the exception `PRINCIPAL_NOT_FOUND_ERROR`, this change will raise this error.